bank/Accounts/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+# Create your models here.
+class Users(models.Model):
+    first_name = models.CharField(max_length=50)
+    second_name = models.CharField(max_length=50)
+    account_name = models.CharField(max_length=100)
+    account_number = models.IntegerField(primary_key=True , unique=True)
+    account_type = models.CharField(max_length=10)
```

Comment:
https://docs.djangoproject.com/en/2.2/ref/models/fields/#choices for implementing choices